test/gui/tst_sharing/test.feature

```diff
@@ -72,21 +135,89 @@ Feature: Sharing
         And user "Brian" has been created on the server with default attributes and without skeleton files
         And user "Alice" has shared folder "simple-folder" on the server with user "Brian" with "all" permissions
         And user "Brian" has set up a client with default settings
-        When the user overwrites the file "simple-folder/textfile.txt" with content "overwrite ownCloud test text file"
-        Then as "Brian" the file "simple-folder/textfile.txt" on the server should have the content "overwrite ownCloud test text file"
+        When the user overwrites the file "Shares/simple-folder/textfile.txt" with content "overwrite ownCloud test text file"
+        Then as "Brian" the file "Shares/simple-folder/textfile.txt" on the server should have the content "overwrite ownCloud test text file"
```

Comment:
Any reason for setting default share location to `Shares` dir? You can avoid many lines of changes if you do noe set the sharing dir.

Reply:
I did it for consistency but now I know the context :smile: